Source/Core/Core/Core.cpp

```diff
@@ -267,6 +267,8 @@ void Stop()  // - Hammertime!
 
 		g_video_backend->Video_ExitLoop();
 	}
+	if (s_emu_thread.joinable())
+		s_emu_thread.join();
```

Comment:
This will cause a deadlock, you aren't allowed to block the UI thread while the GPU thread is still active especially in exclusive mode.